modules/oadp-usecase-restore.adoc

```diff
@@ -0,0 +1,91 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/oadp-use-cases/oadp-usecase-restore-different-namespace.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="oadp-usecase-restore_{context}"]
+= Restoring an application to a different namespace using {oadp-short}
+
+To restore a backup of an application by using {oadp-short}, you create a restore Custom Resource (CR) as shown in the following example. In the example, you restore the backup to a new target namespace, `test-restore-application`. The source namespace refers to the application namespace that you included in the backup. You then verify the restore by changing your project to the new restored namespace and verifying the resources.
```

Comment:
See other comment concerning CR.

Reply:
Done. changed to `custom resource`. Thanks.